modules/ossm-federation-features.adoc

```diff
@@ -0,0 +1,26 @@
+////
+This module included in the following assemblies:
+* service_mesh/v2x/ossm-federation.adoc
+////
+
+[id="ossm-federation-features_{context}"]
+= Federation features
+
+[role="_abstract"]
+Features of the {ProductName} federated approach to joining meshes include the following:
+
+* Supports common root certificates for each mesh.
+* Supports different root certificates for each mesh.
+* Mesh administrators manually configure certificate chains, service discovery endpoints, trust domains, and gateways used for interacting with each mesh in the federation.
+* Only export/import the services that you want to share between meshes.
+** Details about deployed workloads are not visible to other meshes in the federation.
+** A service can be *exported* to make it visible to other meshes, allowing requests from workloads in other meshes in the federation.
+** A service that has been exported by one mesh in the federation can be *imported* by another mesh, enabling its workloads to send requests to the imported service.
+* All communication between meshes is encrypted and utilizes mutual TLS for client verification.
+//* Supports configuring failover for a service to a workload that is deployed in another mesh within the federation.
```

Comment:
Just noting that we discussed this in Slack, and we'll try to include some doc around this use case. We could indicate that it's TP here. There are also some limitations to our failover support that we may want note here as well (or where the use case is described in more detail).